in-toto-spec.md

```diff
@@ -1129,7 +1211,12 @@ metadata:
       "stdin": "",
       "stdout": "",
       "return-value": "0"
-      }
+    },
+    "environment": {
+      "variables": [""],
```

Comment:
why variables has an empty list-like description [""], whereas filesystem and workdir have empty non-list-like description "" ?
(Same comment applies in other places in this document)